mathml/presentation-markup/operators/mo-paint-lspace-rspace.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>&lt;mo&gt; paint lspace rspace</title>
+    <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#tokenmo">
+    <meta name="assert" content="Verifies values for lspace and rspace for element mo in LTR and RTL modes.">
+    <link rel="match" href="mo-paint-lspace-rspace-ref.html">
+  </head>
+  <body>
+    <h1>LTR case</h1>
+
+    <p>The test passes if the arrow has a leading space of 100px, which is as wide as the black block to the left,
+      and a trailing space of 200px, which is as wide as the black block to the right.</p>
+
+    <mth>
```

Comment:
This should have been `<math>` ; I opened https://github.com/web-platform-tests/wpt/pull/16384